rs-matter/src/data_model/objects/handler.rs

```diff
@@ -297,6 +297,18 @@ mod asynch {
     use super::{ChainedHandler, EmptyHandler, Handler, HandlerCompat, NonBlockingHandler};
 
     pub trait AsyncHandler {
+        fn read_awaits(&self, _attr: &AttrDetails) -> bool {
```

Comment:
I think these need documentation - I  was trying to understand the code reading how we use read/write awaits and chunked message handling and I really somehow wanted to understand what various `*_awaits` functions mean. The name is not immediately obvious: a handler tells me something about read/write/invoke. What is it? what does `awaits` mean?

Reply:
Fair enough. A lot of the pre-existing public API is undocumented, and I've started doing this for new code (only). But you are right - `xxx_awaits` is new code too.  I've now documented the whole handler API, and left a couple of TODOs at the places where I expect it to change soon.